lib/factory_bot.rb

```diff
@@ -87,15 +87,22 @@ class << self
              :constructor,
              to: :configuration
 
-    delegate :trait_by_name,
+    delegate :register_sequence,
              :register_trait,
+             :rewind_sequences,
+             :sequence_by_name,
+             :trait_by_name,
              to: Internal
 
     attr_accessor :allow_class_lookup
 
     deprecate :allow_class_lookup,
               :allow_class_lookup=,
               :register_trait,
+              :register_sequence,
```

Comment:
`register_sequence` is part of the public api (see the last section of [the sequence documentation](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences)), so we shouldn't deprecate it.